CMakeLists.txt

```diff
@@ -4,13 +4,17 @@ project(DemoGafferExtension)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_VERBOSE_MAKEFILE OFF)
 
+if(APPLE)
```

Comment:
I think it’s more common to use DARWIN

Reply:
Not on my end

Reply:
oh, ok well if APPLE works lets stick with that